common/commands/errors.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019 IOTA Stiftung
+ * https://github.com/iotaledger/rpchub
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef COMMON_ERRORS_H_
+#define COMMON_ERRORS_H_
+
+#include <map>
+#include <string>
+
+#include <grpc++/support/sync_stream.h>
+#include "grpc++/grpc++.h"
+
+namespace common {
+
+namespace cmd {
+
+typedef enum {
+
+  OK = 0,
+  MISSING_ARGUMENT,
+  WRONG_ARGUMENT_NAME,
+  CANCELLED,
+  USER_EXISTS,
+  USER_DOES_NOT_EXIST,
+  UNKNOWN_ADDRESS,
+  UNKNOWN_ERROR,
+  GET_ADDRESS_FAILED,
+  INVALID_UUID,
+  INVALID_CHECKSUM,
+  INVALID_ADDRESS,
+  ADDRESS_WAS_SPENT,
+  IOTA_NODE_UNAVAILABLE,
+  WITHDRAWAL_CAN_NOT_BE_CANCELLED,
+  INVALID_AUTHENTICATION,
+  SIGNATURE_FAILED,
+  WRONG_USER_ADDRESS,
+  ADDRESS_NOT_KNOWN_TO_NODE,
+  ADDRESS_BALANCE_ZERO,
+  BATCH_INVALID,
+  BATCH_INCONSISTENT,
+  BATCH_AMOUNT_NOT_ZERO,
+  INSUFFICIENT_BALANCE,
+
+} Error;
+
+std::map<Error, std::string> errorToStringMapCreate();
+
+grpc::Status errorToGrpcError(Error err);
+
+static const std::map<Error, std::string> errorToStringMap =
```

Comment:
Maybe extern this and define it in the impl file ? Otherwise everything that include this file will have its own copy of the map.